### PR TITLE
Added my job profile

### PR DESCRIPTION
**Pull request by escapingSamsara**
Please be so kind to add me to the ZtM Job Boad list !

### CODE REVIEW

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
please remove the package-lock.json ... this is not what is currently in the repo. If you would like it to be updated, please create an issue with what the change should be then create a PR tied to the issue.

As for your submission, please remove the @ - otherwise approved

**Review by escapingSamsara — COMMENTED** [review]
i deleted the package json (or didnt i ?)

and a

File: package-lock.json
Comment (ProsperousHeart) [inline package-lock.json:1]: Please remove the deletion of this file from your git. If we approve this, it will be removed from the project completely.

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Aside from removing the package-lock.json from your git, your JSON data is approved.

**Comment by ProsperousHeart** [comment]
> i deleted the package json (or didnt i ?)
> 
> and added the json file without @ and correct user image link

unfortunately deleting it will still update the project when approved and merged. It must be removed from your Git instance

**Comment by ProsperousHeart** [comment]
@escapingSamsara - I have already provided the review. Is there a question about it?

**Pull request closed without merging**